evals/eval_swe_bench.py

```diff
@@ -41,7 +41,7 @@ def check_install_package(package_name):
             return False
 
 
-async def generate_patch_with_cognee(instance, llm_client, search_type=SearchType.CHUNKS):
+async def run_code_graph_pipeline(repo_path):
```

Comment:
this should be in cognee/api/v1/cognify/code_graph_pipeline.py

Reply:
and ../../cognee/notebooks/cognee_code_graph_demo.ipynb should also call this function

Reply:
Great points! I moved the function to `code_graph_pipeline.py` for now. I propose creating a separate issue to clean up that file and update the notebook, though. If you are fine with this please let me know and I will create the issue and resolve this conversation.

Reply:
code_graph_pipeline.py is quite confusing now that it contains both implementations, I wouldn't leave it like this :(
These are the options I see, from least effort to most effort:
- Leave it as is, but add big bold docstrings to the old functions saying "deprecated, use run_code_graph_pipeline instead"
- Remove the old functions, leave the notebook as is, but test the new implementation in test_code_generation.py, otherwise it will break. I would choose this option.
- Remove old functions, update both the notebook and the tests

Reply:
Thanks, @alekszievr! I will go for the least effort option for now, and then we can all sync about it for the next cycle. 
I created [a new issue](https://linear.app/cognee/issue/COG-906/update-cognifycode-graph-pipelinepy-and-related-code) to address this. In the meanwhile, I added some comments and updated the docstring as you suggested.